CoEpi/ui/common/ConfirmationAlert.swift

```diff
@@ -10,16 +10,20 @@ struct ConfirmationAlert {
                                                 message: message,
                                                 preferredStyle: .alert)
 
-        alertController.addAction(UIAlertAction(title: noText, style: .default, handler: { action in
-            viewController.dismiss(animated: true, completion: {
-                noAction()
-            })
+        alertController.addAction(
+            UIAlertAction(title: noText, style: .default, handler: { action in
+                DispatchQueue.main.async {
```

Comment:
Hmm, why do we need this? I just investigated and apparently we don't even need to dismiss the alert? https://stackoverflow.com/questions/30853387/uialertaction-completion-block-not-called-ios/30853553

(I just tested on iOS 13 commenting out `viewController.dismiss(animated: true, completion: nil)` and it was dismissed. Could this maybe fix the issue on iOS 12?)